content/c-sharp/concepts/arrays/terms/resize/resize.md

```diff
@@ -0,0 +1,101 @@
+---
+Title: '.Resize()'
+Description: 'Updates or defines the size of an existing array.'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Arrays'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+The **`.Resize()`** method changes the number of elements of a one-dimensional array to the specified new size.
+
+## Syntax
+
+```pseudo
+// Sets a new size for the specified array
+Array.Resize<int>(ref int[] array, int newSize);
+```
+
+`.Resize()` is a modifier of the `Array` object.  When used as part of a one-dimensional array, it defines the number of elements acceptable by the array, or redefines the size of an array.  This can be used to both lengthen or shorten the number of elements within a given array.
+
+## Example
+
+The following example :
```

Comment:
The introductory sentence should summarize what the example does.

Reply:
This is a part where I definitely had tunnel vision and had forgotten to add the introductory sentence.  A majority of the page's formatting was lifted directly from the other Array pages' formatting.  My apologies for overlooking this on review.  It has since been corrected and updated.